docker/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+set -o errexit
+set -o pipefail
+set -o nounset
+# set -o xtrace
+
+if [ $# -eq 0 ]; then
+    exec coordinator -f test_array.npy --host ${HOST} --port ${PORT}
+    else
+        exec coordinator "$@"
+    esle
```

Comment:
esle?

Reply:
typo on my part for another check I had before. Removing it, good catch!